docs/UserGuide.md

```diff
@@ -120,6 +120,7 @@ Adds a person to the address book.
 
 :bulb: **Tip:**
 A person can have any number of tags (including 0).
+Each tag has a limit of 60 characters.
```

Comment:
Should be under notes since the `Future dates as birthday are not allowed` is under notes too?